Core/Core/SwiftGen/Assets.swift

```diff
@@ -76,9 +76,12 @@ public enum CoreAssets {
   public static let noAnnouncements = ImageAsset(name: "noAnnouncements")
   public static let noHandouts = ImageAsset(name: "noHandouts")
   public static let dashboard = ImageAsset(name: "dashboard")
-  public static let discovery = ImageAsset(name: "discovery")
-  public static let learn = ImageAsset(name: "learn")
-  public static let profile = ImageAsset(name: "profile")
+  public static let discoverActive = ImageAsset(name: "discover_active")
+  public static let discoverInactive = ImageAsset(name: "discover_inactive")
+  public static let learnActive = ImageAsset(name: "learn_active")
+  public static let learnInactive = ImageAsset(name: "learn_inactive")
+  public static let profileActive = ImageAsset(name: "profile_active")
+  public static let profileInactive = ImageAsset(name: "profile_inactive")
   public static let programs = ImageAsset(name: "programs")
```

Comment:
There are two modes of TabView mentioned in `MainScreenView.swift`
- list
- gallery

In list view, we should also add selected and unselected states:
```
.tabItem {
  CoreAssets.programs.swiftUIImage.renderingMode(.template)
  Text(CoreLocalization.Mainscreen.programs)
}
```

If we are not going to maintain list view then we should entirely remove list view related icons and changes.

WDYT?

Reply:
The list view is not our need nor the axim, both providers are using gallery view. If in the future any provider wants to use it, they can configure (open PR) according to their need.